package.json

```diff
@@ -11,15 +11,20 @@
   "scripts": {
     "start": "node app.js",
     "pretest": "npm run lint",
-    "test": "npm run test:unit && npm run test:e2e",
-    "test:e2e": "echo 'no e2e tests available.'",
+    "test": "npm run test:unit",
+    "test:e2e": "npm run selenium:install && npm run selenium:start && npm run nightwatch",
```

Comment:
Might be because I'm running selenium with &?  Maybe that allows npm run to move on?  I need to look into npm run and see how it works